tests/suites/static_analysis/lint_go.sh

```diff
@@ -78,7 +103,9 @@ test_static_analysis_go() {
 
 		cd .. || exit
 
-		run_linter "run_api_imports"
+		run "run_api_imports"
```

Comment:
Should this not be run_linter? AFAIK the benefit of run_linter was to not fail fast. So on CI we get all the failures across all linters.

Reply:
run_linter doesn't work for these tests. I'm not sure why, I had to switch to get it to fail correctly.

Reply:
It's probably because of the pipefail setting.